### PR TITLE
scripts/perftune.py: allow configuring disks write cache mode

### PR DESCRIPTION
**Pull request by vladzcloudius**
Add a new boolean configuration 'write_back_cache'/'--write-back-cache' that would
set the write cache 'write back' setting if set to 'True' or 'write through' if
set to 'False'.

If new parameter is not given - write cache configuraiton would remain unchanged.

Signed-off-by: Vlad Zolotarov <vladz@scylladb.com>

### CODE REVIEW

File: scripts/perftune.py [hunk 1]
```diff
@@ -1294,6 +1317,7 @@ def names():
 argp.add_argument('--options-file', help="configuration YAML file")
 argp.add_argument('--dump-options-file', action='store_true', help="Print the configuration YAML file containing the current configuration")
 argp.add_argument('--dry-run', action='store_true', help="Don't take any action, just recommend what to do.")
+argp.add_argument('--write-back-cache', help="Enable/Disable \'write back\' write cache mode.", dest="set_write_back")
```
Comment (bentsi) [inline scripts/perftune.py:1320]: you can make the default type to _distutils.util.strtobool_ and you won't need to check types after that.
```suggestion
argp.add_argument('--write-back-cache', dest="set_write_back", type=distutils.util.strtobool, default=False, help="Enable/Disable \'write back\' write cache mode.", )
```

Comment (vladzcloudius) [reply scripts/perftune.py:1320]: The whole point is that I don't want to.
Please, read the description of the patch. There is a special meaning when the value is not set: don't change the currently configured write cache configuration.

Defining a default value as you suggested would have changed the desired semantics.

Comment (vladzcloudius) [reply scripts/perftune.py:1320]: @bentsi Please, let me know if that makes more sense now?

**Comment by penberg** [comment]
Merged as https://github.com/scylladb/seastar/commit/1c1f610ceb134b639f54e8f46748982a5d0e9911

**Pull request closed without merging**